Packages/com.unity.inputsystem/InputSystem/Devices/InputDevice.cs

```diff
@@ -187,6 +187,26 @@ public override Type valueType
             get { return typeof(byte[]); }
         }
 
+        public NoiseFilter noiseFilter
```

Comment:
I think if we have this property, the name needs to be refined to point out what the filter applies to. From the name, I would assume that it, well, filters out noise whereas in reality, it doesn't really filter anything but only impacts `current` and doesn't affect what data we store and return.